server/covmanager/SourceCodeProvider/test_sourcecodeprovider.py

```diff
@@ -88,6 +88,7 @@ def runTest(self):
         parents = provider.getParents("c3abaa766d52")
         self.assertEqual(len(parents), 0)
 
+@unittest.skipIf(not os.path.isdir("/home/decoder/Mozilla/repos/mozilla-central-fm"), reason="not decoder")
```

Comment:
( ╯°□°)╯ ┻━━┻ I didn't even mean to check this in. But it's ok to disable it for now until we can properly write a test. And this is a really creative way to do so ;D